chain/client/src/tests/bug_repros.rs

```diff
@@ -315,7 +315,7 @@ fn test_long_gap_between_blocks() {
                             if approval_message.approval.target_height < target_height {
                                 (NetworkResponses::NoResponse.into(), false)
                             } else {
-                                if approval_message.target.as_ref() == "test1" {
+                                if approval_message.target== "test1" {
```

Comment:
Please run linter
```suggestion
                                if approval_message.target == "test1" {
```

Reply:
Please wait before the PR is ready for review.